samcli/hook_packages/terraform/hooks/prepare.py

```diff
@@ -147,7 +150,9 @@ def _update_resources_paths(cfn_resources: Dict[str, Any], terraform_application
     terraform_application_dir: str
         The terraform application root directory where all paths will be relative to it
     """
-    resources_attributes_to_be_updated = {CFN_AWS_LAMBDA_FUNCTION: ["Code"]}
+    resources_attributes_to_be_updated = {
```

Comment:
can you add a unit test case to cover updating the lambda layer paths in the mapped CFN resource dict. check this https://github.com/aws/aws-sam-cli/blob/8c8bc2e54ba1a03b5bc6d0a4cf7bb8d67ea8d3e3/tests/unit/hook_packages/terraform/test_prepare_hook.py#L2150

Reply:
I added some layers to this test since it was already generic enough.